src/status_im/translations/en.cljs

```diff
@@ -254,10 +254,10 @@
    :edit-chats                           "Edit chats"
    :search-chats                         "Search chats"
    :empty-topic                          "Empty topic"
-   :topic-format                         "Wrong format [a-z0-9\\-]+"
-   :topic-name-error                     "A topic name can contain only following characters: a-z -"
+   :topic-format                         "Topic names contain letters and numbers only"
+   :topic-name-error                     "Topic names contain letters and numbers only"
```

Comment:
How does it look in the app when the message is repeated?

Reply:
Maybe I don't understand where these are used. When would they appear together?

Reply:
Do you have a screenshot of these messages in-app? Would be helpful to see both in-context.

Otherwise we have the same message `topic-format` and `topic-name-error` repeated in different places.

Reply:
I've only found `topic-name-error` in the app. I don't know when the other is triggered. 
![topic name error](https://user-images.githubusercontent.com/3373750/44788911-44dc3c80-ab69-11e8-988f-e5702a5d5b7b.png)

Reply:
If we say "Topic names contain letters and numbers only" then upper case letters looks like ok. Same for non-english letters. However, only lowercase english letters, numbers and "-" are valid here. Can we change the copy, please?

Reply:
I'm reluctant to say "English" - how about `Topic names contain only lowercase a through z and dash (-)`

Reply:
or `Topic names use only lowercase letters (a to z) & dashes (-)`

Reply:
ok with Topic names use only lowercase letters (a to z) & dashes (-)